resources/metadata/detect.yaml

```diff
@@ -113,6 +114,35 @@ spec:
         - PARAMETERS
         - STAGES
         - STEPS
+      - name: groups
+        description: Users groups to be assigned for the Project
+        aliases:
+          - name: detect/groups
+        type: '[]string'
+        mandatory: false
+        scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+      - name: failOn
+        description: Mark the current build as fail based the policy categories. A comma seperated list can be provided to fail on multiple categories, for eg. 'BLOCKER,CRITICAL,MAJOR'
```

Comment:
update description: it is a list and not comma separated anymore

Reply:
Thanks. I have updated the description to include the details about the policies and acceptable values